google_drive/public/js/google_calendar.js

```diff
@@ -0,0 +1,18 @@
+/* Javascript for GoogleDocumentBlock. */
+function GoogleCalendarBlock(runtime, element) {
+
```

Comment:
Unnecessary line break here?  Should we include a comment (unsure of generally-accepted JS conventions for documenting functions)